### ISSUE TITLE
Documentation on how to get the authenticated user

### ISSUE THREAD

**Issue by amitassaraf** [hasura/ndc-nodejs-lambda#47]
Hello, currently there seems to be no documentation on how to fetch the JWT token details of the user who has made the request to the typescript function. How can I do this?

**Comment by daniel-chambers** [comment]
If you [configure JWT authentication](https://hasura.io/docs/3.0/auth/authentication/jwt/) with your Supergraph, your claims can be made available as Session Variables. You can then bind session variables to command arguments [using CommandPermissions](https://hasura.io/docs/3.0/data-modeling/permissions#creating-a-commandpermission) ([more docs here](https://hasura.io/docs/3.0/supergraph-modeling/permissions#commandpermissions)). Command arguments are just the arguments you define on your TypeScript function.

So in short:
* Configure JWT auth on your supergraph to get claim values as session variables
* Define arguments that would take your claim values on your functions
* Introspect your connector then add commands for your functions
* Use CommandPermissions to bind the session variables to your arguments
